mava/configs/env/lbf_scenario/Foraging-15x15-4p-3f.yaml

```diff
@@ -0,0 +1,9 @@
+# The config of the Foraging-15x15-4p-3f scenario
+
+task_config_gen:
+  grid_size: 15
+  fov : 15
```

Comment:
A general description somewhere for what these attributes are would be really nice

Reply:
Sure! Also, should add the config and description for the case if we want to use the GridObserver for LBF? (in this case, the observation is a vector and VectorObserver is set as default).